src/js/tabs/cli.js

```diff
@@ -173,6 +201,60 @@ TABS.cli.initialize = function (callback, nwGui) {
         } else {
             $('.tab-cli .copy').hide();
         }
+        
+        $('.tab-cli .load').click(function() {
+            var suffix = 'txt',
```

Comment:
Not sure if we shouldn't be more permissive here - I'd at least add `.config` (which is used for Unified Target configurations).

Reply:
Fair point. I've added the option to show all files or `text/*` files by mime type, in case the user wishes to filter the list of files.

Reply:
Hmm, at least under linux this does not seem to be working properly (not sure what NW.js uses to detect file type):

![image](https://user-images.githubusercontent.com/4742747/56886037-23e63580-6ac2-11e9-8261-846a556fc747.png)

![image](https://user-images.githubusercontent.com/4742747/56886057-2fd1f780-6ac2-11e9-959d-14bcdd095ba1.png)

I think ideally `.config` files should be shown with the default filter when opening the dialog, as this is the extension used for official unified target configurations.

Reply:
Ah, okay, I overlooked that point for first. Here're the new filters force pushed. However, it did work for me on Ubuntu. I presume that on Linux apps should read the mime types from `/etc/mime.types`, but I don't know actually.

Reply:
Good point about `/etc/mime.types`. However, it is a bit feeble for NW.js to only use extensions to 'determine' the type of file on a system that has intrinsic recognition for most common file types:

```
mikeller@pc:~/git/betaflight$ file unified_targets/configs/ALIENFLIGHTF4.config 
unified_targets/configs/ALIENFLIGHTF4.config: ASCII text
```

Reply:
My suggestion would be:
```
var accepts = [
                {
                    description: 'Config files', extensions: ['txt', 'config'],
                },
                {
                    description: 'All files',
                },
            ];
```

- `.txt` is the default extension when saving CLI output;
- `.config` is the extension for official Unified Target configs;
- using the `text/*` seems to be too broad - files with extensions making them unlikely to be valid CLI command sets like `.js`, `.html` are shown.

Reply:
Okay, let it be :)